### DISCUSSION TITLE
Support for  LOCALDEV_DENY_CD

### DISCUSSION THREAD

**Discussion by adityamj** [ltsp/ltsp#266] [Q&A] [answered]
Hi,

In my current deployment of ltsp5 all local storage was restricted. Only a few clients were allowed to access local storage using options like LOCALDEV_DENY_CD.

This option is unavailable as per https://ltsp.org/man/ltsp.conf/

Is there a way to selectively enable local storage on ltsp19?

Thanks,
Aditya

**Comment by alkisg** [comment, marked as answer]
Hi Aditya,

LTSP fat clients (the same in LTSP5 and LTSP20+) behave like regular standalone workstations.
So you just need to google about "restrict disk access Ubuntu" or whatever your distribution is. Hint: /usr/share/polkit-1/actions/org.freedesktop.UDisks2.policy

Normally, only sudoers have access to mount internal disks, and all users have access to removable media like CDs. These can be customized by policykit, but this customization isn't LTSP-specific anymore, like it was with thin clients and LTSPFS.
